integration/specs/DateTimePicker/datetimepicker-1.spec.js

```diff
@@ -0,0 +1,22 @@
+const PageDateTimePicker = require('../../../src/components/DateTimePicker/pageObject');
+
+const DATETIMEPICKER = '#datetimepicker-1';
+
+describe('DateTimePicker', () => {
+    beforeAll(() => {
+        browser.url('/#!/DateTimePicker/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(DATETIMEPICKER);
+        component.waitForExist();
+    });
+    it('should set correct date on modal header when day is clicked', () => {
+        const picker = new PageDateTimePicker(DATETIMEPICKER);
+        picker.click();
+        picker.waitUntilModalIsOpen();
+        expect(picker.getModalHeaderText()).toBe('10/25/19, 10:44 AM');
+        picker.clickDay(10);
+        expect(picker.getModalHeaderText()).toBe('10/10/19, 10:44 AM');
+    });
+});
```

Comment:
also add test cases for:
- should open the DateTimePicker when click on input element
- should open the DateTimePicker when click on the label element (for this create a clickLabel method in page object)
- should close the DateTimePicker when click the ok button (for close test cases you need to add waitUntilClose method in page object)
- should close the DateTimePicker when click the cancel button

we don't need to repeat the modal tests but this above does not belong to modal but to DateTimePicker itself